plugin-build/build.gradle.kts

```
@@ -70,3 +70,41 @@ if (gradle.gradleVersion >= "6.6.0") {
     val publish = extensions.getByType(MavenPublishPluginExtension::class.java)
     publish.releaseSigningEnabled = BuildUtils.shouldSignArtifacts()
 }
+
+apply<DistributionPlugin>()
+
+val sep = File.separator
+
+configure<DistributionContainer> {
+    this.getByName("main").contents {
+        from("build${sep}libs")
+        from("build${sep}publications${sep}maven")
+        from("build${sep}publications${sep}sentryPluginPluginMarkerMaven")
+        duplicatesStrategy = DuplicatesStrategy.INCLUDE
+    }
+}
```

```suggestion
distributions {
    main {
      duplicatesStrategy = DuplicatesStrategy.INCLUDE
      contents {
        from("build${sep}libs")
        from("build${sep}publications${sep}maven")
        from("build${sep}publications${sep}sentryPluginPluginMarkerMaven")
      }
    }
}
```
This should also work.

---

It seems it's not compatible with Gradle 8.0 ([see CI](https://github.com/getsentry/sentry-android-gradle-plugin/runs/3067385850#step:5:302)), so moving back to the previous state.

---

The CI is failing as it's not properly formatted (as I typed that manually 😅). You should be able to solve this by running: `./gradlew -p plugin-build ktlintKotlinScriptFormat` that will reformat the build file.

---

@iker-barriocanal the suggestion makes sense, its kotlin friendlier, lets do it and apply the formatting